desc/equilibrium/utils.py

```diff
@@ -141,6 +141,7 @@ def parse_axis(axis, NFP=1, sym=True, surface=None):
             axis[:, 1],
             axis[:, 2],
             axis[:, 0].astype(int),
+            axis[:, 0].astype(int),
```

Comment:
what is this doing? seems to be passing in the same modes for Rn as Zn?

Reply:
That seems off? or do we ensure before this that we have the full mode spectrum in `n` and just set the non-symmetric modes to zero if it is stell symmetric?